CHANGELOG

```diff
@@ -1,4 +1,30 @@
 # Changelog
+## [0.2.5] - 2020-02-19
+### Breaking changes
+ - Deprecate positional arguments in the following methods:
+   - "libnmstate.apply()"
+   - "libnmstate.commit()"
+   - "libnmstate.rollback()"
+   - "libmstate.show()"
+
+### New Features
+ - Report missing NM plugin during validation.
+ - Support OVS Link Aggregation (LAG).
+
+### Bug Fixes
+ - Fix "AtributeError" when editing an existing OVS interface.
+ - Do not remove slaves implicitly when removing the master interface.
+ - Fix team interface port support.
+ - Prevent virtual interface removal after DHCP or IPV6-RA timeout.
+
+### Known Issues
+
+In nmstate 0.2.5 there are some known issues:
+ - Cannot activate a connection against unmanaged network interfaces.
+ - Cannot reapply when bridge profile previously does not contain 802-3-ethernet section.
```

Comment:
This is too much internal information IMHO. An Nmstate user would not know what this means. I also do not know from this text when exactly this will happen or what this means. I suspect that it means that sometimes the link of interfaces goes down even if it would not be necessary.

Reply:
Should we then remove it or rephrase it? In my opinion, `sometimes the link of interfaces goes down even if it would not be necessary` sounds very generic. Any suggestion? Thanks!

Reply:
@EdDev can you maybe give a high-level description of the bug?

Reply:
How about: `All bridge changes require disruptive actions.` ?

Reply:
Reword: All changes to bridges might disrupt their connectivity